tests/GLX_dummy/GLX_dummy.c

```diff
@@ -664,37 +664,27 @@ static GLboolean dummyInitiatePatch(int type,
     return GL_TRUE;
 }
 
-static void dummyReleasePatch(void)
-{
-}
-
-static const __GLdispatchPatchCallbacks dummyPatchCallbacks =
-{
-    .isPatchSupported = dummyCheckPatchSupported,
-    .initiatePatch = dummyInitiatePatch,
-    .releasePatch = dummyReleasePatch,
-};
 #endif // defined(PATCH_ENTRYPOINTS)
 
-static const __GLXapiImports dummyImports =
+PUBLIC Bool __glx_Main(uint32_t version,
+                                  const __GLXapiExports *exports,
+                                  __GLXvendorInfo *vendor,
+                                  __GLXapiImports *imports)
 {
-    .isScreenSupported = dummyCheckSupportsScreen,
-    .getProcAddress = dummyGetProcAddress,
-    .getDispatchAddress = dummyGetDispatchAddress,
-    .setDispatchIndex = dummySetDispatchIndex,
+    if (version <= GLX_VENDOR_ABI_VERSION) {
```

Comment:
Not sure if I mentioned it previously - we ought to update this to honour the new major/minor version. Otherwise some unsuspecting vendor will do copy it and things will explode down the line.